src/components/Tokens/TokenTable/NetworkFilter.tsx

```diff
@@ -62,7 +64,11 @@ const MenuTimeFlyout = styled.span`
   position: absolute;
   top: 48px;
   z-index: 100;
-  left: 0px;
+  ${({ isInfoExplorePageEnabled }) => (isInfoExplorePageEnabled ? 'right: 0px;' : 'left: 0px;')}
+
+  @media screen and (max-width: ${LARGE_MEDIA_BREAKPOINT}) {
+    ${({ isInfoExplorePageEnabled }) => isInfoExplorePageEnabled && 'left: 0px;'}
```

Comment:
the `isInfoExplorePageEnabled` isn't needed since left is already 0px when not enabled